### DISCUSSION TITLE
Microsoft Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.

### DISCUSSION THREAD

**Discussion by givihuda** [VSCodium/vscodium#1404] [Q&A]
I'm on Windows 10 Pro 64-bit version 21H2 (soon to get Windows 11, I promise), and I was installing GitHub Desktop 64-bit - https://desktop.github.com/ - GitHubDesktopSetup-x64.exe v3.1.6 - and the installation (of Git) let me choose VSCodium as a default editor, which I've never used before, so I clicked the link to install it, found VSCodium-x64-1.75.1.23040.msi, tried to install it, and got the Microsoft Defender error.

![image](https://user-images.githubusercontent.com/74785256/217970936-533ed20c-ca7e-4ba6-8cec-d9e13a1893d9.png)

![image](https://user-images.githubusercontent.com/74785256/217971091-0316b349-0287-43e3-9ab6-2cd73a9ca699.png)

Typically I've used Vim or Notepad++ on Windows, so now I'm wondering of the efficacy of using VSCodium on Windows and big-name apps like GitHub Desktop recommending it if it's a security risk (I know the devs of VSCodium can't speak for GitHub Desktop, but who knows if there's some affiliate deal going on or what that they could speak on).

Anyway, just curious of other's experience or advice here, since I cancelled the install.

I noticed this pull request - https://github.com/VSCodium/vscodium/pull/1246 - that seems to imply that a Microsoft/Windows publisher is already set... honestly, I've never this blue box warning from Windows before; I'm used to the yellow-gray one, so I can't tell if Windows had this updated or it's a special circumstance or something else is going on.

**Comment by daiyam** [comment]
@givihuda You get the warning for any application from publishers without EV certificates.

**Reply by givihuda** [reply]
I've never seen that warning before... that's why I asked.

I've seen this prompt during an install for an unknown publisher:

![image](https://user-images.githubusercontent.com/74785256/221613569-2fa012d6-95d1-4aea-90c4-9458c7f2b10b.png)

That's asking if I want to allow an app from an unknown publisher; that's very different than a prompt telling me Windows Defender protected by PC from an unrecognized app by preventing it from running...

Are you really saying those are the exact same prompts?

Right now I can't tell why I'm getting the blue "protected" prompt for VSCodium when I've only ever seen the yellow "allow" prompt from previous apps install attempts.

**Reply by daiyam** [reply]
More details at https://en.wikipedia.org/wiki/Microsoft_SmartScreen#Windows_malware_protection

**Reply by givihuda** [reply]
Who wrote that 😆 Did you? It seems cynical or critical of Microsoft. 

I mean, I don't consider Microsoft perfect, but I'm also not clear on how you're saying that relates to VS Codium specifically.

Can you relate the exact lines you're saying apply to the issue I've mentioned, and an explanation of how VS Codium applies to that? I think we're just assuming otherwise.

**Reply by daiyam** [reply]
https://www.theregister.com/2020/06/05/windows_10_microsoft_defender_smartscreen/
